test/acceptance/e2e.test.js

```diff
@@ -4,11 +4,27 @@ const util = require('../common/util');
 
 chai.use(require('chai-as-promised'));
 
+const handler = require('serve-handler');
+const http = require('http');
+
 const files = util.loadFiles(path.join(__dirname, 'e2e'));
 const template = util.loadTemplate(path.join(__dirname, 'e2e.html.tpl'));
+const PORT = 5000;
```

Comment:
maybe this should be defined and exported in the `util.js` file

Reply:
:+1: